app/models/concerns/titleable.rb

```diff
@@ -12,6 +12,10 @@ def canonical_title
     titles[self[:canonical_title]]
   end
 
+  def canonical_title_key
```

Comment:
hmmmmm maybe `canonical_title_locale` or something might be better? I'm not a huge fan of just saying `_key` and then not contextualizing what the key _is_

Reply:
👍